x-pack/plugins/cases/docs/openapi/components/schemas/case_response_updated_by_properties.yaml

```diff
@@ -0,0 +1,16 @@
+title: Case response properties for updated_by
```

Comment:
Looks like we repeat the user fields a few times. Would it be possible to store this in a single file and import it to these places?

Reply:
Good point!  I've made that change in 79531d8de1c107f5c20088574f81e1207121998d